modules/rosa-architecture.adoc

```diff
@@ -32,3 +32,15 @@ The following types of clusters are available with ROSA Classic:
 * Single zone cluster - The control plane and worker nodes are hosted on a single availability zone.
 
 * Multi-zone cluster - The control plane is hosted on three availability zones with an option to run worker nodes on one or three availability zones.
+
+Alternatively, you can install a ROSA Classic cluster using AWS PrivateLink, which is hosted on private subnets only.
```

Comment:
This statement does not belong here. 

Few lines ahead already we've listed 3 types of clusters - public, private and private link.